doc/radosgw/config-ref.rst

```diff
@@ -1115,6 +1115,45 @@ Keystone Settings
 :Type: Boolean
 :Default: ``true``
 
+
+
+Object Lifecycle Management Settings
+====================================
+
+
+``rgw enable lc threads``
+
+:Description: Whether the Ceph Object Gateway lifecycle management is enabled.
+:Type: Boolean
+:Default: ``true``
+
+
+``rgw lc lock max time``
+
+:Description: The total run time in seconds for a single lifecycle processor.
+:Type: Integer
+:Default: 60
+
+
+``rgw lc max objs``
+
+:Description: The maximum number of objects that may be handled by
+              lifecycle management thread in one processing cycle.
+              Shoud not be larger than 7877(HASH_PRIME).
```

Comment:
@yehudasa  According to https://github.com/ceph/ceph/blob/master/src/rgw/rgw_lc.cc#L153 
maybe not only for trimming?

what about following:

 ```
The maximum number of rados objects(holding the list of 
buckets that enabled lifecycle management) that be handled by
 lifecycle management thread in one processing cycle.
```